### PR TITLE
feat(data-computer): can choose number of topics with a param

### PR DESCRIPTION
**Pull request by leogail**

### CODE REVIEW

File: services/data-computer/v1/lda-segment.ini [hunk 1]
```diff
@@ -28,6 +28,10 @@ post.parameters.2.name = X-Webhook-Failure
 post.parameters.2.schema.type = string
 post.parameters.2.schema.format = uri
 post.parameters.2.required = false
+post.parameters.3.in = query
+post.parameters.3.name = nbTopic
+post.parameters.3.schema.type = int
```
Comment (parmentf) [inline services/data-computer/v1/lda-segment.ini:33]: **quibble (non-blocking):** ‏I used `number` in [`teeft`](https://github.com/Inist-CNRS/web-services/blob/main/services/terms-extraction/v1/teeft/fr.ini#L24C33-L24C39)

Just check `int` works before merging.

File: services/data-computer/v1/lda.ini [hunk 2]
```diff
@@ -28,6 +28,11 @@ post.parameters.2.name = X-Webhook-Failure
 post.parameters.2.schema.type = string
 post.parameters.2.schema.format = uri
 post.parameters.2.required = false
+post.parameters.3.in = query
+post.parameters.3.name = nbTopic
+post.parameters.3.schema.type = int
```
Comment (parmentf) [inline services/data-computer/v1/lda.ini:33]: **quibble (non-blocking):** ‏Or `number` ?

See `lda-segment.ini`.

File: services/data-computer/v1/lda.py [hunk 3]
```diff
@@ -7,6 +7,9 @@
 import re
 import spacy
 
+
+nbTopic = sys.argv[sys.argv.index('-p') + 1] if '-p' in sys.argv else 15
```
Comment (parmentf) [inline services/data-computer/v1/lda.py:11]: **praise:** ‏Clever

Maybe more readable using a function with a good name?

File: services/data-computer/v1/lda.ini (hunk 2 above)
Comment (leogail) [reply services/data-computer/v1/lda.ini]: done (homogenisation with you)

File: services/data-computer/v1/lda.py (hunk 3 above)
Comment (leogail) [reply services/data-computer/v1/lda.py:11]: Thanks. Just put a comment on this line (is enough i guess)

File: services/data-computer/v1/lda.py [hunk 4]
```diff
@@ -112,6 +115,10 @@ def max_topic(dico):
         sys.stdout.write(json.dumps(line))
         sys.stdout.write("\n")
     else:
+        #### test for stats with prometeus : (2/2)
```
Comment (parmentf) [inline services/data-computer/v1/lda.py:118]:
```suggestion
        #### test for stats with prometheus : (2/2)
```

File: services/data-computer/Dockerfile [hunk 5]
```diff
@@ -28,10 +28,11 @@ RUN pip install \
     pandas==1.4.0 \
     lxml==4.7.1 \
     fr_core_news_sm@https://github.com/explosion/spacy-models/releases/download/fr_core_news_sm-3.6.0/fr_core_news_sm-3.6.0-py3-none-any.whl \
-    sentence-transformers==2.2.2 \
-    umap-learn==0.5.5 \
-    scikit-learn==1.4.1.post1 \
-    scipy==1.10.1
+    # sentence-transformers==2.2.2 \
```
Comment (parmentf) [inline services/data-computer/Dockerfile:31]: **question:** ‏These dependencies were not used ?!?

Are they the reason for the downsizing of the image?

**Review by parmentf — APPROVED** [review]

File: services/data-computer/Dockerfile (hunk 5 above)
Comment (leogail) [reply services/data-computer/Dockerfile]: It is use for clustering. I just delete it for test to build faster

**Pull request merged**